### PR TITLE
BGDIINF_SB-2328: Removing the var DEPLOY_TARGET

### PR DESCRIPTION
**Pull request by procrastinatio**
The var `DEPLOY_TARGET`  was used to build the settings file from templates on a host to  test the settings of the next environment prior to deploy the code to     this environnment (lengthy process), like building on `dev`with settings for `int`. No CI back then :-( and not need anymore with docker.

### CODE REVIEW

File: Makefile.frankfurt [hunk 1]
```diff
@@ -138,21 +144,30 @@ environ:
...
 	envsubst < apache/wsgi-py3.conf.in > apache/wsgi.conf && \
-	envsubst <  apache/application.wsgi.in > apache/application.wsgi && \
+	envsubst < apache/application.wsgi.in > apache/application.wsgi && \
 	envsubst < docker-compose.yml.in > docker-compose.yml && \
 	envsubst < 25-mf-chsdi3.conf.in > 25-mf-chsdi3.conf
 endef
 
+local.ini: local.ini.in base.ini guard-OPENTRANS_API_KEY guard-PGUSER guard-PGPASSWORD guard-GIT_HASH_SHORT guard-APACHE_PORT
+	export CURRENT_DIRECTORY=${CURRENT_DIRECTORY} && \
+	export $(cat .env.default) && \
```
Comment (hansmannj) [inline Makefile.frankfurt:161]: is it correct to hard code this here to cat `.env.default` or would it make sense to change it to cat `ENV_FILE` here? If `ENV_FILE` is correct, probably it makes sense to `guard-` that as well?

File: Makefile.frankfurt [hunk 2]
```diff
@@ -138,21 +144,30 @@ environ:
...
+	${ENVSUBST} <  $< > $@
```
Comment (hansmannj) [inline Makefile.frankfurt:162]: will this only use `local.ini`? Where is `base.ini` used within this target?

**Review by hansmannj — COMMENTED** [review]
Just 2 minor questions

File: Makefile.frankfurt (hunk 2 above)
Comment (procrastinatio) [reply Makefile.frankfurt:162]: `base.ini` is referenced within the `local.ini`

**Review by hansmannj — APPROVED** [review]
Probably a a short comment that .env.default is intentionally hard coded

File: Makefile.frankfurt (hunk 1 above)
Comment (procrastinatio) [reply Makefile.frankfurt]: Commit [4ca402e](https://github.com/geoadmin/mf-chsdi3/pull/3896/commits/4ca402e8343834d2f0a3f1ed4068eaeace4ab222) made it dynamic.

File: Makefile.frankfurt [hunk 3]
```diff
@@ -138,21 +144,36 @@ environ:
 # FIXME: nosetests is still using development.ini
 define build_templates
 	export DOCKER_IMG_LOCAL_TAG=${DOCKER_IMG_LOCAL_TAG} && \
-	export CURRENT_DIR=${CURRENT_DIR} && \
+	export CURRENT_DIRECTORY=${CURRENT_DIRECTORY} && \
 	envsubst < base.ini.in > base.ini && \
 	envsubst < dev.ini.in > dev.ini && \
```
Comment (hansmannj) [inline Makefile.frankfurt:149]: Is the `dev.ini` (line 149) and `development.ini` (line 151) redundant or intention?

File: PYTHON3.md [hunk 4]
```diff
@@ -14,14 +14,18 @@ Install the python virtual environment (still `virtualenv`at this point)
 
     make -f Makefile.frankfurt setup
 
-Set the require environmental variables with
+The required environement variables are set in `.ven.default`
```
Comment (hansmannj) [inline PYTHON3.md:17]: small typo: `.env.default`

File: Makefile.frankfurt [hunk 5]
```diff
@@ -156,9 +156,15 @@ define build_templates
...
 endef
 
+define setup_env
+	@echo "Setup vars from ENV_FILE=$(ENV_FILE)"
+	$(eval include $(1))
+	$(eval export sed 's/=.*//' $(1))
+endef
+
 local.ini: local.ini.in base.ini guard-OPENTRANS_API_KEY guard-PGUSER guard-PGPASSWORD guard-GIT_HASH_SHORT guard-APACHE_PORT
+	$(call setup_env,$(ENV_FILE))
```
Comment (hansmannj) [inline Makefile.frankfurt:166]: Would a guard-ENV_FILE make sense in that case?

Comment (procrastinatio) [reply Makefile.frankfurt:166]: Yes.

**Pull request merged**